lv_conf_template.h

```diff
@@ -83,6 +83,12 @@
     #endif
 #endif  /*LV_USE_STDLIB_MALLOC == LV_STDLIB_BUILTIN*/
 
+/*Layer memory allocation limits*/
+#define LV_USE_DRAW_LAYER_MAX_MEMORY 0
```

Comment:
It's not required, because this feature can be disabled by setting a very large value for `LV_DRAW_LAYER_MAX_MEMORY`.

Reply:
Yes, but IMO setting a flag for global enable is a clearer way of signaling we dont want to limit things at all.

Reply:
We should target having as few configs as possible. `lv_coonf.h` is already too long and complicated. 
We if can save one here, we should. :slightly_smiling_face:

Reply:
Okay, I can see the point you made. I'll change it.